ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -60,6 +66,14 @@ def uninstall_cluster_logging():
         pvc_obj.delete(resource_name=pvc_list['items'][pvc]['metadata']['name'])
         pvc_obj.wait_for_delete(resource_name=pvc_list['items'][pvc]['metadata']['name'])
 
```

Comment:
Also please verify that pv does not exists.

Reply:
done

Reply:
Please add this check before verifying the image deletion.